modules/network-observability-lokistack-ingestion-query.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+:_content-type: PROCEDURE
+[id="network-observability-lokistack-configuring-ingestion{context}"]
+
+= Configuring LokiStack ingestion
+The LokiStack instance comes with default settings according to the configured size. It is possible to override some of these settings, such as the ingestion and query limits. You might want to update them if you get Loki errors showing up in the Console plugin, or in `flowlogs-pipeline` logs.
+
+Here is an example of configured limits:
+
+[source,yaml]
+----
+spec:
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 40
+        ingestionRate: 20
+        maxGlobalStreamsPerTenant: 25000
+      queries:
+        maxChunksPerQuery: 2000000
+        maxEntriesLimitPerQuery: 10000
+        maxQuerySeries: 3000
+----
+Refer to the LokiStack API reference for more information on these settings.
+
+A good practice is to define an alert, to get notified when these limits are reached. In the example below, the alert uses a metric provided by the Loki operator, `loki_request_duration_seconds_count`:
+
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: loki-alerts
+  namespace: openshift-operators-redhat
+spec:
+  groups:
+  - name: LokiRateLimitAlerts
+	rules:
+	- alert: LokiTenantRateLimit
+  	annotations:
+    	message: |-
+      	{{ $labels.job }} {{ $labels.route }} is experiencing 429 errors.
+    	summary: "At any number of requests are responded with the rate limit error code."
+  	expr: sum(irate(loki_request_duration_seconds_count{status_code="429"}[1m])) by (job, namespace, route) / sum(irate(loki_request_duration_seconds_count[1m])) by (job, namespace, route) * 100 > 0
+  	for: 10s
+  	labels:
+    	severity: warning
+----
```

Comment:
Seems like there was indentation lost during the copy. Cf the original: https://github.com/netobserv/documents/blob/main/examples/distributed-loki/alerting/loki-ratelimit-alert.yaml

```suggestion
----
apiVersion: monitoring.coreos.com/v1
kind: PrometheusRule
metadata:
  name: loki-alerts
  namespace: openshift-operators-redhat
spec:
  groups:
  - name: LokiRateLimitAlerts
    rules:
    - alert: LokiTenantRateLimit
      annotations:
        message: |-
          {{ $labels.job }} {{ $labels.route }} is experiencing 429 errors.
        summary: "At any number of requests are responded with the rate limit error code."
      expr: sum(irate(loki_request_duration_seconds_count{status_code="429"}[1m])) by (job, namespace, route) / sum(irate(loki_request_duration_seconds_count[1m])) by (job, namespace, route) * 100 > 0
      for: 10s
      labels:
        severity: warning
----
```